src/components/Primer/contants.ts

```diff
@@ -0,0 +1,2 @@
+export const CONTAINER = 'msd__primerWrapper';
```

Comment:
```suggestion
export const CONTAINER_CLASS_NAME = 'msd__primerWrapper';
```

also, would be good to replace `CONTAINER` with something more specific